src/markdown-pages/docs/cli-command-reference/runs-prepare.md

```diff
@@ -11,35 +11,55 @@ The following section provides a subset of examples that use the `runs prepare`
 
 ### Getting help
 
-Use the following commands to get more information about a command and command options, including default values.
+Use the following commands on Unix or Mac to get more information about a command and command options, including default values.
 
 ```
 galasactl --help
-galasactl runs --help
-galasactl runs prepare --help
+```
+
+For Windows, use 
+
+```
+galasactl.exe --help
 ```
 
 ### Selecting tests from a test steam
 
-The following example command selects tests from a test stream called `BestSoFar` for inclusion in a portfolio called `test.yaml`. All tests in the `dev.galasa.simbank.tests`  package within the `BestSoFar` test stream are added to the `test.yaml` portfolio. Packages are selected if the name contains a specified string or matches the regex if --regex is specified.  
+The following example command selects tests from a test stream called `BestSoFar` for inclusion in a portfolio called `test.yaml` if you are using Mac or Unix. All tests in the `dev.galasa.simbank.tests`  package within the `BestSoFar` test stream are added to the `test.yaml` portfolio. Packages are selected if the name contains a specified string or matches the regex if --regex is specified.  
 
 ```
-galasactl runs prepare
-                        --portfolio test.yaml
-                        --stream BestSoFar
+galasactl runs prepare \
+                        --portfolio test.yaml \
+                        --stream BestSoFar \
+                        --package dev.galasa.simbank.tests
+```
+
+If you are using Windows, use the following example command:
+```
+galasactl.exe runs prepare `
+                        --portfolio test.yaml `
+                        --stream BestSoFar `
                         --package dev.galasa.simbank.tests
 ```
 
 ### Selecting tests without a test stream
 
 You can use test class names to build a portfolio when a test stream or test catalog is not available. 
 
-The following example command selects specified tests (`SimBankIVT` and `BasicAccountCreditTest`) from the `dev.galasa.simbank.tests` test package and adds these tests to the `test.yaml` portfolio. 
+The following example command selects specified tests (`SimBankIVT` and `BasicAccountCreditTest`) from the `dev.galasa.simbank.tests` test package and adds these tests to the `test.yaml` portfolio on Mac or Unix. 
+
+```
+galasactl runs prepare \
+                        --portfolio test.yaml \
+                        --class dev.galasa.simbank.tests/SimBankIVT \
+                        --class dev.galasa.simbank.tests/BasicAccountCreditTest
+```
 
+If you are using Windows, use the following example command:
 ```
-galasactl runs prepare
-                        --portfolio test.yaml
-                        --class dev.galasa.simbank.tests/SimBankIVT
+galasactl.exe runs prepare `
```

Comment:
Seems good to have these both documented. +1

Reply:
done